test/runtime/Policies.go

```diff
@@ -1015,6 +1031,119 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
 			failCurl := vm.ContainerExec(helpers.App1, helpers.CurlFail("http://%s/public", httpd2[helpers.IPv4]))
 			failCurl.ExpectFail("unexpectedly able to access %s when access should only be allowed to host", helpers.Httpd2)
 		})
+
+		// In this test we rely on the hostDockerContainer serving on a
+		// secondary IP, which is otherwise not bound to an identity to
+		// begin with; it would otherwise be part of the cluster. When
+		// we define CIDR policy on it, Cilium allocates an identity
+		// for it.
+		testCIDRL4Policy := func(policy, dstIP, proto string) {
+			_, err := vm.PolicyRenderAndImport(policy)
+			ExpectWithOffset(1, err).To(BeNil(), "Unable to import policy: %s", err)
```

Comment:
This `err` in the assert message is redundant, the `err` message is already in the assert.